apps/ensadmin/src/components/providers/react-query-client.ts

```diff
@@ -0,0 +1,48 @@
+import { QueryClient, defaultShouldDehydrateQuery, isServer } from "@tanstack/react-query";
+
+/**
+ * Create a query client to be used on the server and browser.
+ *
+ * @returns A query client.
+ */
+function makeQueryClient() {
+  return new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: 10 * 1000,
```

Comment:
Not sure what is the right value here. Assuming this one will prevent queries from being refetched client-side for 10 seconds.